src/fdc3/dotnet/DesktopAgent/src/MorganStanley.ComposeUI.DesktopAgent/Infrastructure/Internal/Constants.cs

```diff
@@ -0,0 +1,29 @@
+﻿/*
+* Morgan Stanley makes this available to you under the Apache License,
+* Version 2.0 (the "License"). You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0.
+*
+* See the NOTICE file distributed with this work for additional information
+* regarding copyright ownership. Unless required by applicable law or agreed
+* to in writing, software distributed under the License is distributed on an
+* "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+* or implied. See the License for the specific language governing permissions
+* and limitations under the License.
+*/
+
+using System.Reflection;
+
+namespace MorganStanley.ComposeUI.Fdc3.DesktopAgent.Infrastructure.Internal;
+
+internal static class Constants
+{
+    public static string DesktopAgentProvider = "ComposeUI";
+    public static string SupportedFdc3Version = "2.0.0";
+    public static string? ComposeUIVersion = Assembly.GetExecutingAssembly().GetName().Version?.ToString();
```

Comment:
No point in using GetExecutingAssembly here, better to use typeof(Constants).Assembly (or any other type from the assembly)
See: https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.getexecutingassembly?view=net-8.0#remarks

Not that this is a performance-wise critical area, but it's a good/bad practice thing.